test/mix/tasks/phoenix.new_test.exs

```diff
@@ -75,6 +75,22 @@ defmodule Mix.Tasks.Phoenix.NewTest do
     end
   end
 
+  test "bootstraps generated project with adapter option" do
```

Comment:
This isn't running a new application (nor it should) so it should not be here. There are tests in the installer directory and just asserting on the generated content there should be enough.